lib/localdb.js

```diff
@@ -30,7 +30,7 @@ function getDitchHandle(cb) {
       my_db_config = parseMongoConnectionURL(mongoConnectionURL);
     }
     catch (e) {
-      var errMessage = new Error("Incorrect format for database connection string.");
+      var errMessage = new Error(`Error parsing database connection string - verify format. ${e}`);
```

Comment:
Added this to include the specific error detail. I had a failing unit test due to a typo and this kept printing "Error parsing database connection string" without the actual error 😄